common/fe-ste-models.go

```diff
@@ -589,40 +589,40 @@ func (ft *FromTo) FromAndTo(s string) (srcLocation, dstLocation Location, err er
 	return
 }
 
-func (ft *FromTo) To() Location {
-	return Location(((1 << 8) - 1) & *ft)
+func (ft FromTo) To() Location {
+	return Location(((1 << 8) - 1) & ft)
 }
 
-func (ft *FromTo) From() Location {
-	return Location((((1 << 16) - 1) & *ft) >> 8)
+func (ft FromTo) From() Location {
+	return Location((((1 << 16) - 1) & ft) >> 8)
 }
 
-func (ft *FromTo) IsDownload() bool {
+func (ft FromTo) IsDownload() bool {
 	return ft.From().IsRemote() && ft.To().IsLocal() && ft.To() != ELocation.None() && ft.To() != ELocation.Unknown()
 }
 
-func (ft *FromTo) IsS2S() bool {
+func (ft FromTo) IsS2S() bool {
 	return ft.From().IsRemote() && ft.To().IsRemote() && ft.To() != ELocation.None() && ft.To() != ELocation.Unknown()
 }
 
-func (ft *FromTo) IsUpload() bool {
+func (ft FromTo) IsUpload() bool {
 	return ft.From().IsLocal() && ft.To().IsRemote() && ft.To() != ELocation.None() && ft.To() != ELocation.Unknown()
 }
 
-func (ft *FromTo) IsDelete() bool {
+func (ft FromTo) IsDelete() bool {
 	return ft.To() == ELocation.Unknown()
 }
 
 func (ft *FromTo) IsSetProperties() bool {
```

Comment:
should we change this one as well to have the signature (ft FromTo) since it is not modifying any actual property

Reply:
Probably yes.